README.md

```diff
@@ -27,6 +27,40 @@ meson test -C build
 
 "No tests defined." is printed if the required version of googletest could not be found.
 
+# Configuration
+
+The daemon reads configuration from `/etc/user-identification-manager.conf`. A default configuration
+will be used if the file does not exist. Another path can be used by passing the `-c`/`--config`
```

Comment:
What default configuration? Add link to file or description

Reply:
"Content that corresponds to default configuration with comments explaining the variables" is written just below. Close enough or do you think a "(see below)" is needed?

Reply:
Sorry, missed it. Good enough